youtube/fdm_pl.po

```diff
@@ -10,12 +10,12 @@ msgstr ""
 
 msgctxt "QObject|"
 msgid "YouTube Channel"
-msgstr ""
+msgstr "Kanał YouTube"
 
 msgctxt "QObject|"
 msgid "YouTube Playlist"
-msgstr ""
+msgstr "Lista utworów YouTubec"
```

Comment:
Hi! Are you sure "YouTubec" is right translation?

Reply:
My fault.
Can you change to "Lista utworów YouTube" ?

Reply:
Ok, sure.